RereleaseTimer/utils.js

```diff
@@ -0,0 +1,41 @@
+const SESSION_DURATIONS = ["30s-1min", "1-30min", "30min-1h"];
+
+module.exports = {
+    getLatestRelease: function(releases, sourceGroup) {
+
+        // "releases" contains an array with one latest release for each distribution group.
+        for (release of releases) {
+            if (isInGroup(release, sourceGroup)) {
+                return release;
```

Comment:
Is that possible that we can have 2 release in the group, in this case - should we return only the first one?

Reply:
Can't be - this is how it comes from the App Center https://openapi.appcenter.ms/#/distribute/releases_listLatest2